src/tools/quemap/main.c

```diff
@@ -525,7 +483,7 @@ int32_t main(int32_t argc, char **argv) {
 		Com_Error(ERR_FATAL, "No action specified. Try %s --help\n", Com_Argv(0));
 	}
```

Comment:
Technically no since Com_Error causes an exit() in every code path, but I'll add one anyways.